native/cocos/bindings/event/EventDispatcher.cpp

```diff
@@ -423,17 +422,15 @@ void EventDispatcher::dispatchDestroyWindowEvent(cc::ISystemWindow *window) {
 
 void EventDispatcher::dispatchRecreateWindowEvent() {
 #if CC_PLATFORM == CC_PLATFORM_WINDOWS
-    EventDispatcher::dispatchCustomEvent(EVENT_RECREATE_WINDOW, 1,
-                                         reinterpret_cast<void *>(CC_GET_MAIN_SYSTEM_WINDOW()->getWindowHandle()));
+    EventDispatcher::dispatchCustomEvent(EVENT_RECREATE_WINDOW, 1, ISystemWindow::mainWindowId);
 #else
     EventDispatcher::dispatchCustomEvent(EVENT_RECREATE_WINDOW, 0);
 #endif
 }
 
 void EventDispatcher::dispatchRecreateWindowEvent(cc::ISystemWindow *window) {
```

Comment:
Whether it is possible to consider merging the two versions of dispatchRecreateWindowEvent

Reply:
It should be possible, the previous overloading was for compatibility.